certstrap/config-production.yaml

```diff
@@ -0,0 +1,14 @@
+---
+src-repo: https://github.com/square/certstrap
+src-branch: master
+src-private-key: ~
```

Comment:
Nope, that's why it's `~`. It's only relevant if it ends up being a private one at some point (i.e. a fork).